.github/CONTRIBUTING.md

```diff
@@ -32,15 +32,48 @@ traefik*
 
 - You need `go` v1.8+
 - It is recommended you clone Træfik into a directory like `~/go/src/github.com/containous/traefik` (This is the official golang workspace hierarchy, and will allow dependencies to resolve properly)
-- This will allow your `GOPATH` and `PATH` variable to be set to `~/go` via:
+- Set your `GOPATH` and `PATH` variable to be set to `~/go` via:
```

Comment:
I agree 👍 

Maybe we could mention the fact that it is possible to defined a custom GOPATH.

Maybe add this link: https://golang.org/doc/go1.8#gopath